### ISSUE TITLE
The paper's results could not be replicated

### ISSUE THREAD

**Issue by EleFreelance** [JirongZhang/DeepHomography#56]
![image](https://github.com/JirongZhang/DeepHomography/assets/32585199/173d99ef-5e9f-4848-8100-b43838a4d9de)
![image](https://github.com/JirongZhang/DeepHomography/assets/32585199/2e162ea3-72d1-4779-8037-63554c1d6cc7)

Although my training loss has converged very well, the test error remains significantly high.
why? I synchronized the reduction of both batch size and learning rate. batch_size=16, lr=0.000025. and  train 1000k iteration.
